frontend/packages/console-app/src/components/modals/restore-pvc/restore-pvc-modal.tsx

```diff
@@ -0,0 +1,255 @@
+import './_restore-pvc-modal.scss';
+
+import * as React from 'react';
```

Comment:
This should move up before the scss file importing

Reply:
ack